### PR TITLE
Add Flatpak for wavbreaker

### PR DESCRIPTION
**Pull request by thp**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions: Wayland + X11 for GUI, PulseAudio for libao output, filesystem home for file read/write access to audio files
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains http://wavbreaker.sourceforge.net
- [x] Any additional patches or files have been submitted to the upstream projects concerned: Yes, the packaging is based on [what we have in upstream](https://github.com/thp/wavbreaker/blob/master/scripts/flatpak/net.sourceforge.wavbreaker.json) 

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: net.sourceforge.wavbreaker.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "app-id": "net.sourceforge.wavbreaker",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
```
Comment (hfiguiere) [inline net.sourceforge.wavbreaker.json:4]:
```suggestion
    "runtime-version": "21.08",
```

File: net.sourceforge.wavbreaker.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "sdk": "org.freedesktop.Sdk",
+    "command": "wavbreaker",
+    "modules": [
+        {
+            "name": "libao",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/xiph/libao.git"
```
Comment (hfiguiere) [inline net.sourceforge.wavbreaker.json:14]: you need a specific commit / tag

File: net.sourceforge.wavbreaker.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/thp/wavbreaker/archive/refs/tags/0.14.tar.gz",
+                    "sha256": "ad6ad4dda8c1e435e8dbe22d30176bc54130b1120fda335c1776493535ed3f43"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```
Comment (hfiguiere) [inline net.sourceforge.wavbreaker.json:30]: finish-args is usually located above `modules` by convention.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
I also see that the meson build want mpg123 but it's not listed in the manifest.

**Comment by hfiguiere** [comment]
bot, build net.sourceforge.wavbreaker

**Comment by hfiguiere** [comment]
also it's missing an appstream file. 

(this will cause the build to fail)

**Comment by thp** [comment]
> I also see that the meson build want mpg123 but it's not listed in the manifest.

As far as I understand it, `mpg123` is part of the freedesktop SDK, and can be re-used:

https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/elements/components/mpg123.bst

**Comment by thp** [comment]
> also it's missing an appstream file.

New upstream version 0.15 now installs an appstream file (https://github.com/thp/wavbreaker/commit/d169e81de3c1c8b78461ac927b0e87ee5ee73d9a).

**Comment by hfiguiere** [comment]
bot, build net.sourceforge.wavbreaker

**Comment by barthalion** [comment]
(On unrelated note @thp, I have used urlwatch for many years as Arch developer to monitor new upstream releases. Thanks for developing it!)

/merge

**Pull request closed without merging**